lib/components/DatePicker.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+    <input 
+        :max="maxDate" 
+        :min="minDate" 
+        @change="dateChange($event)" 
+        :value="calcDate" 
+        type="date"
+    />
+</template>
+
+<script>
+export default {
+    name: 'DatePicker',
+    props: [
+        'model',
+        'day',
+        'month',
+        'year',
+        'minDate',
+        'maxDate',
+        'dateChange'
+    ],
```

Comment:
Still need v-model binding here right?

Reply:
Yes, i managed to change the value only by using the event called dateChange which takes an event parameter and then assings the target value.